cmd/nats-server-config-reloader/main.go

```diff
@@ -59,13 +60,15 @@ func main() {
 	fs.Var(&fileSet, "config", "NATS Server Config File (may be repeated to specify more than one)")
 	fs.IntVar(&nconfig.MaxRetries, "max-retries", 5, "Max attempts to trigger reload")
 	fs.IntVar(&nconfig.RetryWaitSecs, "retry-wait-secs", 2, "Time to back off when reloading fails before retrying")
+	fs.IntVar(&customSignal, "signal", 1, "Signal to send to the NATS Server process")
```

Comment:
Maybe we should say "Signal to send to the NATS Server process (default SIGHUP 1)".